File: GoEnv/go/common/EdocDistCert/Test/BlabEaddrGetIndividualClientTest.go
package main

import (
	blabClient "../Client"
	testClient "./TestConfig"
	"log"
)

// 공인전자주소 조회 테스트(개인)
func main() {
	log.SetFlags(log.LstdFlags | log.Lshortfile)
	err := testClient.BlabTestClientInit()
	if err != nil {
		return
	}

	clientConfig := testClient.ClientConfig
	appRes, err := blabClient.GetEaddr(clientConfig.Server.BaseUrl, clientConfig.Individual.Idn, clientConfig.Auth.PlatformId)
	if err != nil {
		log.Printf("err=%+v\n", err)
		return
	}

	log.Printf("appRes=%+v", appRes)
	log.Printf("appRes.Data=%+v", appRes.Data)
}

File: GoEnv/go/common/EdocDistCert/Test/node/BlabEaddrGetIndividualClientTest.js
const { BlabClient } = require('./BlabClient');
const { BlabClientConfig } = require('./BlabClientConfig');

// 개인 공인전자주소 조회 테스트
async function BlabEaddrGetIndividualTest() {
    const res = await BlabClient.GetEaddr(BlabClientConfig.server.baseUrl,
        BlabClientConfig.individual.idn, BlabClientConfig.auth.platformId);
    console.log(res)
}

BlabEaddrGetIndividualTest();